Sources/Fingertips/FingerTipWindow.swift

```diff
@@ -11,20 +11,10 @@ class FingerTipView: UIImageView {
 }
 
 @objc (MBXFingerTipOverlayWindow)
-class FingerTipOverlayWindow: UIWindow {
-    override var rootViewController: UIViewController? {
-        set {
-            super.rootViewController = newValue
-        }
-
-        get {
-            return FingerTipWindow.fingerTipWindow?.rootViewController ?? super.rootViewController
-        }
-    }
-}
```

Comment:
This was causing a warning in logs; the getter also looks suspicious - sharing a view controller between UIWindows sounds dangerous. I don't see any issue with removing this code.